src/styles/main.scss

```diff
@@ -10,6 +10,21 @@
   --title-font: 'Tsukimi Rounded', sans-serif;
 }
 
+// Mobile breakpoint: 575px;
+// Medium breakpoint: 768px;
+// Desktop breakpoint: 1200px;
+
+@media (min-width: 769px) {
+  #root {
+    display: flex;
+  }
+
+  .main {
+    flex: 1;
+    padding: 1rem;
```

Comment:
Would it make sense not to add this padding to the main class? This would conflict with the pages that have a button coming out of the navbar border.

![image](https://github.com/DavidEspinoG/mc_videogames_front/assets/70329406/19dc59ad-9ae0-4e09-9906-ad0bb934c5de)
![image](https://github.com/DavidEspinoG/mc_videogames_front/assets/70329406/b2fb9883-e9fb-4c9a-9e47-579b69258139)